openhexa/sdk/pipelines/utils.py

```diff
@@ -118,7 +120,8 @@ def get_local_workspace_config(path: Path):
                 # S3 connection
                 elif connection_config["type"] == "s3":
                     try:
-                        env_vars[f"{stringcase.constcase(slug)}_SECRET_ACCESS_KEY"] = connection_config[
```

Comment:
@qgerome  I found that there's a mismatch between the S3Connection class on the SDK and the backend : on the SDK we have `SECRET_ACCESS_KEY` and on the Backend  `ACCESS_KEY_SECRET`. So that's why I've updated the mapping here.